src/volt/semantic/extyper.d

```diff
@@ -2506,7 +2506,7 @@ ir.Type extypeBinOp(Context ctx, ref ir.Exp exp, Parent parent)
 
 	final switch (binop.op) with (ir.BinOp.Op) {
 	case Equal, NotEqual, Less, LessEqual, Greater, GreaterEqual:
-		if (!willConvert(ltype, rtype)) {
+		if (!willConvert(ltype, rtype, /*ignore const*/true)) {
```

Comment:
For a later change (aka add a task on trello) if we want to go that way, add a enum. Because bools are bad mmmkay.